src/main/java/uk/gov/companieshouse/overseasentitiesapi/utils/ComparisonHelper.java

```diff
@@ -129,7 +129,16 @@ public static boolean equals(PersonName personName, String string) {
             return nullValuesCheck;
         }
 
-        return personName.toString().equals(string);
+        return StringUtils.equalsIgnoreCase(normalise(personName.toString()),normalise(string));
```

Comment:
Not sure we need this one? I though the problem was with FormerNames?

Reply:
As used - just need to check other uses are not affected.

Reply:
> Not sure we need this one? I though the problem was with FormerNames?
This was added so that it ignores case if the surname is uppercased...